packages/contracts/src/dollar/core/StakingShare.sol

```diff
@@ -34,37 +30,38 @@ contract StakingShareForDiamond is ERC1155, ERC1155Burnable, ERC1155Pausable {
     uint256 private _totalSupply;
 
     // ----------- Modifiers -----------
-    modifier onlyMinter() {
+    modifier onlyMinter() override {
         require(
-            accessCtrl.hasRole(GOVERNANCE_TOKEN_MINTER_ROLE, msg.sender),
-            "Governance token: not minter"
+            accessCtrl.hasRole(STAKING_SHARE_MINTER_ROLE, msg.sender),
+            "Staking Share: not minter"
         );
         _;
     }
 
-    modifier onlyBurner() {
+    modifier onlyBurner() override {
         require(
-            accessCtrl.hasRole(GOVERNANCE_TOKEN_BURNER_ROLE, msg.sender),
-            "Governance token: not burner"
+            accessCtrl.hasRole(STAKING_SHARE_BURNER_ROLE, msg.sender),
```

Comment:
Is it redundant to include `_ROLE` in the role names?

Reply:
it is a constant so if you name it STAKING_SHARE_BURNER it refers to an address